<ui version="4.0" >
 <class>TableWidgetMultMatchTestFrameClass</class>
 <widget class="QFrame" name="TableWidgetMultMatchTestFrameClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TableWidgetMultMatchTestFrame</string>
  </property>
  <widget class="QPushButton" name="button_Select" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>420</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>select</string>
   </property>
  </widget>
  <widget class="BasarTableWidget" name="tableWidget" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>761</width>
     <height>181</height>
    </rect>
   </property>
   <property name="rowCount" >
    <number>2</number>
   </property>
   <property name="columnCount" >
    <number>5</number>
   </property>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="BasarTableWidget" name="tableWidget_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>220</y>
     <width>761</width>
     <height>181</height>
    </rect>
   </property>
   <property name="rowCount" >
    <number>2</number>
   </property>
   <property name="columnCount" >
    <number>5</number>
   </property>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="TableWidgetMultMatchSelectFrame" name="frameMatcherSelect1" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>410</y>
     <width>311</width>
     <height>351</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="TableWidgetMultMatchSelectFrame" name="frameMatcherSelect2" >
   <property name="geometry" >
    <rect>
     <x>490</x>
     <y>410</y>
     <width>301</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="button_shownonsence" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>470</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>show nonsence</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>530</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>op 152 Test 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>570</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>op 152 Test 2</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>BasarTableWidget</class>
   <extends>QTableWidget</extends>
   <header>libbasarqtwidget_tablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>TableWidgetMultMatchSelectFrame</class>
   <extends>QFrame</extends>
   <header>tablewidgetmultmatchselectframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
